pageserver/src/walingest.rs

```diff
@@ -463,6 +464,9 @@ impl WalIngest {
                 if info == pg_constants::XLOG_RUNNING_XACTS {
                     let xlrec = crate::walrecord::XlRunningXacts::decode(&mut buf);
                     self.checkpoint.oldestActiveXid = xlrec.oldest_running_xid;
+                    todo!() // checkpoint_modified=true missing?
```

Comment:
@lubennikovaav , you added this code in https://github.com/neondatabase/neon/issues/6705